### DISCUSSION TITLE
No se visualizan tareas dentro de SMs de cliente gestión documental

### DISCUSSION THREAD

**Discussion by DavidEhrenfeld** [Cotalker/documentation#746] [4. Bug Reports] [answered]
Desde el 9/3/2022 los canales existentes para cliente de gestión documental, no estaban desplegando ninguna tarea creada dentro de cada SM. Estas tareas incluyen notas que contienen los procedimientos que el cliente utiliza y al no visualizarlos, no puede utilizar Cotalker, siendo un tema crítico.

**Comment by Yiakman** [comment, marked as answer]
David, ya quedó corregido. Tuvo que ver con intentar habilitar Google Sign In sin tener una cuenta de prueba. De momento, deshabilitamos Google Sign In hasta que podamos probar la integración
